README.md

```diff
@@ -46,12 +46,15 @@ You can configure the bundle in your `config.yml`. Full Example:
 webfactory_piwik:
     # Required, no default. Must be set to the site id found in the Matomo control panel
     site_id: 1
-    # Required, has default. Usually, you only want to include the tracking code in a production environment
-    disabled: '%kernel.debug%'
     # Required. no default. Hostname and path to the Matomo host.
     piwik_host: my.piwik.hostname
-    # Required, has default. Path to the tracking script on the host.
+    # Optional, has default. Usually, you only want to include the tracking code in a production environment
+    disabled: '%kernel.debug%'
+    # Optional, has default. Path to the tracking script on the host.
     tracker_path: "/js/"
+    # Optional, has default. Disable cookies in favor of GDPR
```

Comment:
I'd note here that this currently is not required and has a default value of "false", but that the default will change in the next major version.